YLTableViewTests/YLTableViewDataSourceTests.m

```diff
@@ -0,0 +1,94 @@
+//
+//  YLTableViewDataSourceTests.m
+//  YLTableView
+//
+//  Created by Ushhud Khalid on 12/9/15.
+//  Copyright © 2015 Yelp. All rights reserved.
+//
+
+#import "YLTableView.h"
+#import "YLTableViewBaseTestCase.h"
+#import "YLTableViewCell.h"
+#import "YLTableViewCellEstimatedRowHeight.h"
+#import "YLTableViewCellTestStub.h"
+#import "YLTableViewDataSource.h"
+#import "YLTableViewDataSourceTestStub.h"
+
+@interface YLTableViewDataSourceTests : YLTableViewBaseTestCase
+
+@property (strong, nonatomic) YLTableView *tableView;
+@property (strong, nonatomic) YLTableViewDataSourceTestStub *dataSource;
+
+@property (strong, nonatomic) NSIndexPath *indexPathOfConformingCell;
+@property (strong, nonatomic) NSIndexPath *indexPathOfNonConformingCell;
+
+@end
+
+@implementation YLTableViewDataSourceTests
+
+#pragma mark - Setup/Teardown
+
+-(void)setUp {
+  [super setUp];
+
+  _indexPathOfConformingCell = [NSIndexPath indexPathForRow:0 inSection:0];
+  _indexPathOfNonConformingCell = [NSIndexPath indexPathForRow:1 inSection:0];
+  
+  _tableView = [[YLTableView alloc] init];
+  _dataSource = [[YLTableViewDataSourceTestStub alloc] init];
+  _tableView.dataSource = _dataSource;
+  _tableView.delegate = _dataSource;
+  
+  _dataSource.tableViewCells = [self _cellsForTableView];
+}
+
+-(void)tearDown {
+  _tableView.delegate = nil;
```

Comment:
i don't think you need to do this, if the ```@property``` references are cleaned up across test runs